mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,15 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a, b, c) {
+return a + b + c;
+}
+function introduceMe(name, age) {
+  return "Hello, my name is " + '${name}' + "and I am " + '${age}' + " years old";
```

Comment:
Will `'${name}'` just print out the name?

Reply:
i don t know